src/compas/geometry/distance.py

```diff
@@ -371,6 +371,63 @@ def distance_point_plane(point, plane):
     return fabs(dot_vectors(vector, normal))
```

Comment:
Maybe consider changing `distance_point_plane` to simply call `distance_point_plane_signed` and return the `fabs` of it, so:
```python
def distance_point_plane(point, plane):
    return fabs(distance_point_plane_signed(point, plane))

```